yoti-sdk-api/src/main/java/com/yoti/api/client/identity/constraint/SourceConstraint.java

```diff
@@ -10,14 +10,16 @@
 
 public class SourceConstraint implements Constraint {
 
+    private static final String SOURCE_CONSTRAINT = "SOURCE";
+
     @JsonProperty(Property.TYPE)
-    private static final String type = ConstraintConstants.SOURCE_CONSTRAINT;
+    private final String type = SOURCE_CONSTRAINT;
```

Comment:
Avoid field initialisation mixed with ctor initialisation. Assign the value within the ctor (as it's easier to read if everything is initialised in the same "place")